app/src/utils/analytics.ts

```diff
@@ -1,7 +1,8 @@
-import { metrics } from '@sentry/nextjs'
-import { track } from '@vercel/analytics/react'
+import { db, setDoc, doc } from '@/config/firebase'
```

Comment:
NIT but wouldn't mind if we import it like this:

```suggestion
import * as Firebase from '@/config/firebase'
```

So that the call sites are more contextual and informative without needing to click around to understand what's happening.

Reply:
Fixed in 001a60f74d076b2f3f26781968876633bfb0960a. Why do have a lint rule that specifically disallows this though if it's encouraged in the codebase?

Reply:
I never encouraged this 👀 Nuno is taking over right now haha

Reply:
In fact, I am strongly against this :)

Reply:
@noahjoeris can you explain why you are strongly against this?

Reply:
Tbh, I also prefer explicit imports unless there are namespace clashes or a lot of stuff to import. Tree-shaking is one benefit, IDEs adding explicit imports via hot-key is another.

fa1de5900344f95406e4aaece37fb59a7d09170f

Reply:
Yeah that's fine, let's go with that 🙌🏻